packages/rum/src/domain/segmentCollection/segmentCollection.spec.ts

```diff
@@ -124,27 +117,11 @@ describe('startSegmentCollection', () => {
     })
 
     it('flushes segment when the page become hidden', () => {
-      setPageVisibility('hidden')
-      const { addRecord, eventEmitter, sendCurrentSegment, worker, sendSpy } = startSegmentCollection(CONTEXT)
-      addRecord(RECORD)
-      eventEmitter.dispatchEvent(createNewEvent(DOM_EVENT.VISIBILITY_CHANGE))
-
-      worker.processAllMessages()
-      expect(sendSpy).toHaveBeenCalledTimes(1)
+      const { lifeCycle, sendCurrentSegment } = startSegmentCollection(CONTEXT)
+      lifeCycle.notify(LifeCycleEventType.PAGE_EXITED, { isUnloading: false })
       expect(sendCurrentSegment().creation_reason).toBe('visibility_hidden')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏here, we are deducing that a page exit that is not unloading is a visibility hidden, maybe we could use an exit reason instead of a boolean